libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -18,7 +18,7 @@
 #define VELNE_NOISE_DEFAULT     0.5f
 #define VELD_NOISE_DEFAULT      0.7f
 #define POSNE_NOISE_DEFAULT     1.0f
-#define ALT_NOISE_DEFAULT       1.5f
+#define ALT_NOISE_DEFAULT       2.0f
```

Comment:
Did your testing show that this noise increase is needed for all 3 platforms evenly at 2.0?